docs/Functions.md

```diff
@@ -614,9 +614,11 @@ This function has been removed. The dependency that makes this function work is
 You can continue using the function by manually installing the [`files_scripts_deprecated`](https://github.com/Raudius/files_scripts_deprecated) app, which bundles all the removed functions.
 ### mustache
 
-`mustache(String template, [Table variables]={}): String`  
+`mustache(String template, [Table variables]={}, [Bool escape]=true): String`  
```

Comment:
The default value is actually `false`

Also see other change request; you should leave this file unchanged and just make the docs changes directly in the class file comment.

Reply:
Oh yep, I think copilot did that and I missed it. I've only added a couple of lines so feel free to add it in your master branch however you want if it's something you'd like to add :)